.github/workflows/test.yml

```diff
@@ -16,7 +16,7 @@ jobs:
       matrix:
         arch:
           - armhf
-          - mips
+          - mipsel
 
     container:
       image: debian:buster
```

Comment:
The pull request mentions updating it to Debian bullseye, but it's still using buster?

Reply:
That was pretty stupid of me. It was supposed to be a temporary change to find out the reason why qemu stopped working. Thanks.

- https://github.com/hg/tinc/actions/runs/1146226549
- https://github.com/hg/tinc/actions/runs/1146251417